src/SummaryCell.tsx

```diff
@@ -16,10 +17,10 @@ function SummaryCell<R, SR>({
 }: SummaryCellProps<R, SR>) {
   const { summaryFormatter: SummaryFormatter, summaryCellClass } = column;
   const className = clsx(
```

Comment:
I wonder if we should create a helper component that can render the default styles for example

```
export function DefaultCell({ column, className, props }) {
className = clsx(
    cellClassname,
    {
      [cellFrozenClassname]: column.frozen,
      [cellFrozenLastClassname]: column.isLastFrozenColumn
    },
    className
  );

  return (
    <div
      role="gridcell"
      aria-colindex={column.idx + 1}
      className={className}
      {...props}
    >
     {children}
    </div>
  );
}
```

Wdyt?

Reply:
It's more stuff for react to render, not sure 🤔